README.md

```diff
@@ -915,8 +915,10 @@ See also [About Rust’s Machine Learning Community](https://medium.com/@autumn_
 
 ### Asynchronous
 
+
```

Comment:
This is why the Markdown lint is failing

Reply:
Do I need to do something?